Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -40,21 +43,35 @@ def write_data(self, message):
         else:
             self.events_request.write(message)
 
+    def start_new_query(self):
+        self.infohashes_sent = set()
```

Comment:
No, line 29/30 only triggers when the API starts but we need to reset the set of info hashes sent every time we perform a new query (since the same torrent might be in the result set of two queries).